### PR TITLE
Update contributors, address typos

### PR DESCRIPTION
**Pull request by meli-lewis**
Roping in Roberto because I know he's also working on the main README! I'm also happy to update it today because I'll be poking around all day regardless.

### CODE REVIEW

File: project2/README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+This section covers scraping pages on the Web with Python using a [get](http://www.w3schools.com/tags/ref_httpmethods.asp) request.
 
-To begin with, we will import the [modules](http://docs.python.org/3/tutorial/modules.html) or packages we need to scrape websites. This is done with a series of import statements at the top of the file. 
+To begin with, we will import the [modules](http://docs.python.org/3/tutorial/modules.html) or packages we need to scrape websites. This is done with a series of import statements at the top of the file.
 
 ```Python
 import requests
 from bs4 import BeautifulSoup
-import csv
+import CSV
```
Comment (robroc) [inline project2/README.md]: Are you sure the csv module is in upper case?

File: project2/README.md [hunk 2]
```diff
@@ -54,25 +54,25 @@ At this point we are going to stop and take a minute to prepare the file we will
...
 
-The `with` statement is a convenient way Python offers to handle opening and closeing files and cleaning up memory. This saves us from having to close the file later and deal with any other garbage collection.
+The `with` statement is a convenient way Python offers to handle opening and closing files and cleaning up memory. This saves us from having to close the file later and deal with any other garbage collection.
 
-_NOTE: any code that writes to this file must be indented when using `with`. For this reason, the code in the Jupyter notebook won't use notation because it splits the code into several chunks. We're including thi here becaue it's good practice for the future._
+_NOTE: any code that writes to this file must be indented when using `with`. For this reason, the code in the Jupyter notebook won't use notation because it splits the code into several chunks. We're including this here because it's good practice for the future._
 
 Here we create and open a file named "movies.csv". The ```"w"``` tells Python that we want it to open the file in write mode. Other options are read ```"r"``` and append to a file ```"ab"```.
```
Comment (robroc) [inline project2/README.md]: Oops, we should also remove the `b` from `"ab"` so it's consistent.

**Comment by robroc** [comment]
Good catch on all those typos. I really should proofread my work...

File: project2/README.md (hunk 1 above)
Comment (tommeagher) [reply project2/README.md]: Agreed. import CSV will throw a ModuleNotFoundError

File: project2/README.md [hunk 3]
```diff
@@ -54,25 +54,25 @@ At this point we are going to stop and take a minute to prepare the file we will
 
 ```Python
 
-with open("movies.csv", "w", newline="") as csvfile:
-	output = csv.writer(csvfile, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL)
+with open("movies.csv", "w", newline="") as CSVfile:
+	output = CSV.writer(csvfile, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL)
```
Comment (tommeagher) [inline project2/README.md]: yes, this CSV won't work, unless we `import csv as CSV` which may be even more confusing for the newcomers.

**Pull request merged**